model/generate.go

```diff
@@ -219,7 +219,11 @@ func (g *GeneratedProject) saveNewBuildsAndTasks(ctx context.Context, cachedProj
 			p.BuildVariants[i].Tasks[j].Priority = t.Priority
 		}
 	}
-
+	// for patches activate all builds, otherwise activate ones that are not setting batchtime
+	var variantsToActivate []string
+	if !evergreen.IsPatchRequester(v.Requester) {
```

Comment:
Should this also case on AdhocRequester? (I'm seeing it where we look at batchtime in the repotracker but I'm only guessing what it means)

Reply:
Good catch, fixed. Ad hoc is when you submit just a config and tell us to run tasks so it's similar to a patch